frontend/src/components/Summary/ImplementationRow.js

```diff
@@ -0,0 +1,162 @@
+import "./ImplementationRow.css";
+import Svg from "../../assets/svg/Svg";
+
+const ImplementationRow = ({ lines, implementation, index }) => {
+  const implementationArray = lines.filter((element) => element.implementation);
+  const caseArray = lines.filter((element) => element.case);
+  //   const {
```

Comment:
Looks like there's some dead code here.

Reply:
oops! I was trying to use React's context API to pass the data to the parent component but then used the state-lifting method. Either way we can do that. Btw removed the dead code.